vector/src/main/java/im/vector/activity/VectorHomeActivity.java

```diff
@@ -1310,7 +1423,7 @@ public void run() {
      * @return fab view
      */
     public FloatingActionButton getFloatingActionButton() {
-        return mFloatingActionButton;
+        return null;
```

Comment:
This remove a feature: the FAB is automatically hidden during a few seconds when the user scroll the page.

Reply:
I gracefully swapped it out for a View. Since only the 'Top' of the button is queried for.